minigalaxy/ui/preferences.py

```diff
@@ -98,6 +100,10 @@ def save_pressed(self, button):
         Config.set("keep_installers", self.switch_keep_installers.get_active())
         Config.set("stay_logged_in", self.switch_stay_logged_in.get_active())
 
+        if self.switch_show_hidden_games.get_active() != Config.get("show_hidden_games"):
+            Config.set("show_hidden_games", self.switch_show_hidden_games.get_active())
+            self.parent.reset_library()
```

Comment:
Once implementation is done in __filter_library_func() we will not need this "if" condition and reset_library(). Just filter_library() instead.